### DISCUSSION TITLE
Afficher et paramétrer AgendApp

### DISCUSSION THREAD

**Discussion by ABINolan** [witsa/synapps#800] [Q&A] [answered]
Bonjour,
Est-t-il possible d'afficher et de rendre paramétrable un agenda dans Synapps.
Malheuresement le détail de reflet ne me permet pas cela. 
Merci !
![image](https://github.com/user-attachments/assets/89a97d67-5ebc-4d38-a116-7a0004296276)

**Comment by manufitoussiwit** [comment, marked as answer]
Bonjour, 

Oui, il faut passer par l'acteur Agenda.

**Reply by ABINolan** [reply]
Merci cela fonctionne, j'avais pourtant essayé il suffisais de patienter pour voir l'agenda apparaître !
